cmd/image-processing/main_test.go

```diff
@@ -7,77 +7,74 @@ package main_test
 import (
 	"context"
 	"fmt"
-	"io"
 	"log"
+	"net/http/httptest"
+	"net/url"
 	"os"
 	"strconv"
 
-	"github.com/google/go-containerregistry/pkg/authn"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/shipwright-io/build/cmd/image-processing"
+
 	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/google/go-containerregistry/pkg/registry"
 	containerreg "github.com/google/go-containerregistry/pkg/v1"
 	"github.com/google/go-containerregistry/pkg/v1/empty"
 	"github.com/google/go-containerregistry/pkg/v1/remote"
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
 	"github.com/spf13/pflag"
-
-	. "github.com/shipwright-io/build/cmd/image-processing"
+	"k8s.io/apimachinery/pkg/util/rand"
 )
 
 var _ = Describe("Image Processing Resource", func() {
 	run := func(args ...string) error {
-		log.SetOutput(io.Discard)
+		log.SetOutput(GinkgoWriter)
```

Comment:
Let me know what you folks think: With Ginkgo (2) running in verbose mode, you now get structured tests with the respective output nicely framed into the output. Without verbose, the usage of the `GinkgoWriter` ensures we do not spoil standard or error output.